.github/workflows/tests.yaml

```diff
@@ -9,6 +9,7 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      fail-fast: false
```

Comment:
Tests for all specified versions run either way, but once the first one for 3.9-3.11 finishes, the other ones get cancelled. Unintuitive as it might be, this seems to fix that behaviour and makes them actually pass

Reply:
This is strange and should be investigated. It should only be cancelled if there is a failure in one of the jobs in the matrix. There could be a failure in some step (e.g. upload to coveralls) even if the tests pass. Or maybe you detected a "glitch in the matrix"?

Reply:
Might have been a glitch: https://github.com/ghga-de/hexkit/actions/runs/9257275581/attempts/2
Somehow 3.9 completed successfully and failed at the same time, so this can probably be removed again

Reply:
I removed the line